doc/source/dsintro.rst

```diff
@@ -366,13 +366,56 @@ dictionary.
 
 .. _basics.dataframe.from_series:
 
-From a Series
-~~~~~~~~~~~~~
+From a Pandas Series
+~~~~~~~~~~~~~~~~~~~~
 
 The result will be a DataFrame with the same index as the input Series, and
 with one column whose name is the original name of the Series (only if no other
 column name provided).
 
+.. ipython:: python
+
+   df = pd.DataFrame()
+   df
+   x1 = pd.Series([1,2,3,4], index=[1,2,3,4])
+   x1
+   df['A'] = x1
+   df
+   x2 = pd.Series([5,5], index=[1, 3])
+   x2
+   df['B'] = x2
+   df
+   lst = [9, 9, 9, 9]
+   x3 = pd.Series(lst, df.index)
+   x3
+   x4 = df['C']
+   df['D'] = pd.Series(df['C']+1, df.index)
+   df
+
+From a Numpy Series
+~~~~~~~~~~~~~~~~~~~
+
+Numpy series is list or list of lists with no index.
+
+.. ipython:: python
+
+    x1 = np.random.randint(0, 10, (5))
+    x1
+    x2 = np.random.randint(-1.0, 10, (5))
+    x2
+    df1 = pd.DataFrame(x1, columns=['A'])
```

Comment:
yeah i would rather have these examples in the doc-string of the Series/DataFrame constructor than here